frontend/src/components/BCDataGrid/components/Editors/SimpleEditor.jsx

```diff
@@ -0,0 +1,41 @@
+import { forwardRef, useEffect, useRef } from 'react'
+
+export const SimpleEditor = forwardRef(
+  ({ value, onValueChange, eventKey, rowIndex, column }, ref) => {
+    const updateValue = (val) => {
+      onValueChange(val === '' ? null : val)
+    }
+
+    useEffect(() => {
```

Comment:
minor optimization, untested

```jsx

    useEffect(() => {
      // Initialize the input value and focus on the input field
      let startValue = value || ''

      if (eventKey === 'Backspace') {
        startValue = ''
      } else if (eventKey && eventKey.length === 1) {
        startValue = eventKey
      }

      updateValue(startValue)

      // Focus on the input field only if it's not already focused
      if (refInput.current && document.activeElement !== refInput.current) {
        refInput.current.focus()
      }
    }, [])

Reply:
deleting this component as it hasn't been used for any of our usecases.